amundsen_application/api/metadata/v0.py

```diff
@@ -168,13 +169,18 @@ def update_table_owner() -> Response:
         else:
             raise Exception('method not handled')
 
+        results_dict = _get_table_metadata(table_key=table_key, index=0, source="")
```

Comment:
We should not be making a request here. I see this information is being used just to get the table data, for the purpose of getting the schema, tablename, and database. I also see a hardcoded Lyft specific url. You should be able to do the following:
1. It seems we should put the `table_name` and `schema` into the request json so that we can grab it from the there.
2. The `table_key` should already what we need to create the `resource_url`
3. Create a config variable for the url with a default value of `localhost:5000`, and in our internal logic will be assigned accordingly. This logic should use that config variable instead of hardcoding anything.